src/parse/import.rs

```diff
@@ -58,41 +64,7 @@ fn find_import(file_path: &PathBuf, name: &OsStr, load_paths: &[&Path]) -> Optio
 }
 
 impl<'a> Parser<'a> {
-    pub(super) fn import(&mut self) -> SassResult<Vec<Stmt>> {
-        self.whitespace();
-
-        match self.toks.peek() {
-            Some(Token { kind: '\'', .. })
-            | Some(Token { kind: '"', .. })
-            | Some(Token { kind: 'u', .. }) => {}
-            Some(Token { pos, .. }) => return Err(("Expected string.", *pos).into()),
-            None => return Err(("expected more input.", self.span_before).into()),
-        };
-
-        let Spanned {
-            node: file_name_as_value,
-            span,
-        } = self.parse_value(true)?;
-        let file_name = match file_name_as_value {
-            Value::String(s, QuoteKind::Quoted) => {
-                if s.ends_with(".css") || s.starts_with("http://") || s.starts_with("https://") {
-                    return Ok(vec![Stmt::Import(format!("\"{}\"", s))]);
-                } else {
-                    s
-                }
-            }
-            Value::String(s, QuoteKind::None) => {
-                if s.starts_with("url(") {
-                    return Ok(vec![Stmt::Import(s)]);
-                } else {
-                    s
-                }
-            }
-            _ => return Err(("Expected string.", span).into()),
-        };
-
-        self.whitespace();
-
+    pub fn _parse_single_import(&mut self, file_name: &str, span: Span) -> SassResult<Vec<Stmt>> {
```

Comment:
Rather than starting this function with an underscore, let's just make it private by removing `pub`.